x/feemarket/keeper/migrations.go

```diff
@@ -11,3 +17,15 @@ func NewMigrator(keeper Keeper) Migrator {
 		keeper: keeper,
 	}
 }
+
+func (m Migrator) Migrate1to2(ctx sdk.Context) error {
+	store := ctx.KVStore(m.keeper.storeKey)
+	baseFeeKeyPrefix := []byte{2}
```

Comment:
use constants from types

Reply:
the constant is removed from types already.